src/coreclr/tools/ILCompiler-DependencyGraph-Viewer/ILCompiler-DependencyGraph-Viewer.csproj

```
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(NetCoreAppToolCurrent)-windows</TargetFramework>
+    <UseWindowsForms>true</UseWindowsForms>
+    <OutputType>Exe</OutputType>
+    <RootNamespace>DependencyLogViewer</RootNamespace>
+    <AssemblyName>ILCompiler-DependencyGraph-Viewer</AssemblyName>
+
+    <ApplicationManifest>app.manifest</ApplicationManifest>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Diagnostics.Tracing.TraceEvent">
+      <Version>$(TraceEventVersion)</Version>
+    </PackageReference>  </ItemGroup>
```

nit, indentation is a bit off:
```suggestion
  <ItemGroup>
    <PackageReference Include="Microsoft.Diagnostics.Tracing.TraceEvent" Version="$(TraceEventVersion)" />
  </ItemGroup>
```

---

Thanks. I've fixed this.